ibis-server/README.md

```diff
@@ -1,16 +1,59 @@
 # Ibis server
 
-## Environment Setup
+## Quick Start
+### Running on Docker
+Pull the image from the GitHub Container Registry
+```bash
+docker pull ghcr.io/canner/wren-engine-ibis:latest
+```
+Create `.env` file and fill in the environment variables (see [Environment Variables](#Environment-Variables)) \
+```bash
+vim .env
+```
+Run the container
+```bash
+docker run --env-file .env -p 8000:8000 ghcr.io/canner/wren-engine-ibis:latest
+```
+### Running on Local
+Requirements:
+- Python 3.11
+- [poetry](https://github.com/python-poetry/poetry) 1.7.1
+
+Clone the repository and navigate to the ibis directory
+```bash
+git clone git@github.com:Canner/wren-engine.git
+cd ibis-server
+```
+Create `.env` file and fill in the environment variables (see [Environment Variables](#Environment-Variables))
+```bash
+vim .env
+```
+Install the dependencies
+```bash
+make install
+```
+Run the server
+```bash
+make run
+```
+
+## Contributing
```

Comment:
```suggestion
## Developer Guide
```
It's a kind of guide to show how start a development environment for developers.
I think `Developer Guide` would be better.